virtwho/virt/rhevm/rhevm.py

```diff
@@ -50,6 +49,47 @@
 }
 
 
+class RhevmConfigSection(VirtConfigSection):
+    """
+    This class is used for validation of RHEVM virtualization backend
+    section. It tries to validate options and combination of options that
+    are specific for this virtualization backend.
+    """
+
+    VIRT_TYPE = 'rhevm'
+
+    def __init__(self, section_name, wrapper, *args, **kwargs):
+        super(RhevmConfigSection, self).__init__(section_name, wrapper, *args, **kwargs)
+
```

Comment:
I think the following is necessary to ensure 'server', 'username' and 'password' (or the encrypted variety) is provided:
```
        self.add_key('server', validation_method=self._validate_server, required=True)
        self.add_key('username', validation_method=self._validate_username, required=True)
        self.add_key('password', validation_method=self._validate_unencrypted_password, required=True)
```